cfy_manager/components/usage_collector/usage_collector.py

```diff
@@ -98,6 +97,15 @@ def _deploy_collector_scripts(self):
             common.chown(constants.CLOUDIFY_USER,
                          constants.CLOUDIFY_GROUP,
                          destination_path)
+
+        utils_script_name = 'script_utils.py'
+        utils_src_path = join(constants.COMPONENTS_DIR,
+                              USAGE_COLLECTOR,
+                              SCRIPTS,
+                              utils_script_name)
+        utils_dest_path = join(SCRIPTS_DESTINATION_PATH, utils_script_name)
+        files.deploy(utils_src_path, utils_dest_path)
```

Comment:
After the deploy I think you should also `chmod` and `chown` as the other files.
Maybe we can add 2 functions, one for deploying the utils and second for the chmod and chown.